src/lib/plugins/system-info.js

```diff
@@ -0,0 +1,517 @@
+// This library contains several chunks of code from Sebastian Hildebrandt's
+// systeminformation library, MIT
+// https://github.com/sebhildebrandt/systeminformation
```

Comment:
That was my initial idea but the library seems to be not compatible with our Babel setup so Webpack launch a lot of compilation errors.

The library also have support for a lot of stuff we doesn't need (almost 2000 lines without docs or lines breaks), personally I would not use it as a dependency.

So I made my own light version but wanted to keep some recognition for the project.